source/sessions/snapshot-sessions.rst

```diff
@@ -130,19 +130,19 @@ Note that the ``snapshot`` property is optional. The default value of
 this property is false.
 
 Snapshot reads and causal consistency are mutually exclusive. Therefore if ``snapshot`` is set to true,
-``causalConsistency`` property is set to false. Client MUST throw an Error if both ``snapshot`` and ``causalConsistency`` are set to true.
-Snapshot reads are supported both on primaries and secondaries.
+``causalConsistency`` must be false. Client MUST throw an error if both ``snapshot`` and ``causalConsistency`` are set to true.
+Snapshot reads are supported on both primaries and secondaries.
 
-ClientSession changes
+ClientSession Changes
 =====================
 
 Transaction are not allowed with snapshot sessions.
-Calling ``session.startTransaction(options)`` on snapshot session should raise an error.
+Calling ``session.startTransaction(options)`` on a snapshot session SHOULD raise an error.
```

Comment:
Not sure why the original text wasn't using "MUST" but I opted just to capitalize the term to avoid any functional changes with this PR.

Reply:
Do you mind changing this to "MUST"?

Reply:
```suggestion
Calling ``session.startTransaction(options)`` on a snapshot session MUST raise an error.
```